taraxa/state/dpos/precompiled/dpos_contract.go

```diff
@@ -479,6 +479,30 @@ func (self *Contract) DistributeRewards(rewardsStats *rewards_stats.RewardsStats
 	}
 }
 
+func (self *Contract) delegate_check_amount(ctx vm.CallFrame, validator *Validator) error {
+	if self.cfg.ValidatorMaximumStake.Cmp(bigutil.Add(ctx.Value, validator.TotalStake)) == -1 {
```

Comment:
aren't you missing `self.cfg.ValidatorMaximumStake.Cmp(bigutil.Big0) != 0` part? that means there is no MaxValue

Reply:
We have this check in init. There is no need to check it on every delegation

Reply:
ok, I did not know that